llama_index/schema.py

```diff
@@ -387,6 +387,8 @@ class ImageNode(TextNode):
     # TODO: store reference instead of actual image
     # base64 encoded image str
     image: Optional[str] = None
+    image_path: Optional[str] = None
```

Comment:
one thing is that we in fact can get `image_path` from metadata.
I prefer that we do this way to explicitly have image_path as an attribution for image node
and metadata can store another copy.
but there will be some duplicated info for each image node

Reply:
The problem with storing it in metadata is it could be stored under any string.

So, instead we add an attribute to ensure typed access. I think it's fine to duplicate the paths for now. Future loaders for images should set these fields as needed